tests/package.json

```diff
@@ -3,15 +3,16 @@
   "name": "tests_for_theia-apps",
   "version": "0.0.1",
   "scripts": {
-    "test": "wdio wdio.conf.js --theia-port 4000",
-    "test-app-theia": "wdio wdio.theia-docker.conf.js --theia-port 4000",
+    "test": "wdio wdio.conf.js --theia-port 3000",
```

Comment:
When we run CI on Travis, running the Theia apps on whatever port we chose would be ok. 

However we originally chose 4000 because when running tests locally, Theia contributors will often have many instances running on their machine, and a collision is likely when using default port 3000 (and 3001, 3002,...). 

Can you roll-back to port 4000?

Reply:
Reverted